config/locales/translations/zendesk_apps_support.yml

```diff
@@ -182,6 +182,11 @@ parts:
       key: "txt.apps.admin.error.app_build.oauth_keys.missing.other"
       title: "App builder job: missing oauth fields error"
       value: "Missing required oauth fields in manifest: %{missing_keys}"
+  - translation:
+      key: "txt.apps.admin.error.app_build.too_many_oauth_parameters"
+      title: "App builder job: too_many_oauth_parameters"
```

Comment:
Can you please specify 'oauth' as an authorisation standard and is not to be translated.
Suggestion: "App builder job: upload error when there are more than one 'oauth' parameters. 'oauth' is an authorisation standard and is not to be translated"

Reply:
@ovillegaszen Hi Oscar! I have changed to your suggestion :)

Reply:
@zendesk/localization Hi Localization Team! Could someone have a look at this for us please?